web/src/sagas/index.js

```diff
@@ -36,9 +42,13 @@ function* fetchZoneHistory(action) {
 }
 
 function* fetchGridData(action) {
-  const { datetime } = action.payload;
+  const { datetime, timescale } = action.payload;
   try {
-    const payload = yield call(protectedJsonRequest, datetime ? `/v3/state?datetime=${datetime}` : '/v3/state');
+    let path = datetime ? `/v3/state?datetime=${datetime}` : '/v3/state';
+    if (timescale !== TIMESCALE.LIVE) {
+      path += `?timescale=${timescale}_${moment().startOf('month').subtract(1, 'month').format('YYYYMM')}`;
```

Comment:
Why is the `timescale` value necessary here (given that the `datetime` info is already passed)? :thinking: 

If this is to tell the backend to return the values averaged over a month, perhaps I'd prefer to go with something like `datetime=2020-07-01T00:00:00Z&avg_over=1mo`, but I'm not so sure myself :smile:

Reply:
By the way, this endpoint doesn't really work for me (while the history one does), I get an empty map and this message from `state?timescale=MONTHLY_202006`:

![Screenshot from 2020-07-13 17-06-44](https://user-images.githubusercontent.com/1216874/87320496-4c254d00-c52b-11ea-8c4b-421a925e9cdb.png)

Reply:
For your first question, indeed it is to tell the backend that the values should be averaged over 1 month.
For the second, it is because the backend is currently not updating itself by populating the monthly averaged values.
We'll need to do a bit more work to get there :)